components/brave_wallet_ui/panel/async/wallet_panel_async_handler.ts

```diff
@@ -293,6 +293,9 @@ handler.on(PanelActions.approveHardwareTransaction.getType(), async (store: Stor
       await store.dispatch(PanelActions.setSelectedTransaction(txInfo))
       await store.dispatch(PanelActions.navigateTo('transactionDetails'))
       apiProxy.panelHandler.setCloseOnDeactivate(true)
+      // By default the focus is moved to the browser window automatically
```

Comment:
nit: Suggest explaining a bit clearer here:
By default the focus is moved to the browser window automatically when Trezor popup closed which triggers an OnDeactivate event that would close the wallet panel because of the above API call. However, there could be times that the above call happen after OnDeactivate event, so the wallet panel would stay open after Trezor popup closed. As a workaround, we manually set the focus back to wallet panel here so it would trigger another OnDeactivate event when user clicks outside of the wallet panel.